documentation/openapi-authoring-automated-guidelines.md

```diff
@@ -2032,13 +2030,13 @@ Links: [Index](#index) | [Error vs. Warning](#error-vs-warning) | [Automated Rul
 
 ### <a name="r3024" ></a>R3024 EnumUniqueValue
 
-**Category** : ARM Warning
+**Category** : SDK Error
 
 **Applies to** : ARM and Data Plane OpenAPI(swagger) specs
 
 **Output Message** : Enum must not contain case-insensitive duplicated value and make sure every value in enum unique.
```

Comment:
Output message can be short as below:

Enum must not contain duplicated value (case insensitive).

Reply:
I prefer contain more info  in output message